dataland-frontend/src/components/forms/parts/fields/PercentageFormField.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="form-field">
+    <UploadFormHeader :name="`${displayName} (%)`" :explanation="info" :is-required="required" />
+    <FormKit
+      type="text"
```

Comment:
Again just out of curiosity:  Can't FormKit accept an actual number for cases like this? Does that have downsides?

Reply:
Broken: In Firefox I can type "L" into the "representedEmployees" field and submit that.
It is NOT actually submitted, but looks like it to the user (Successful Upload badge!)